app/packages/core/src/components/ColorModal/ColorFooter.tsx

```diff
@@ -47,7 +52,7 @@ const ColorFooter: React.FC = () => {
           text={"Reset"}
           title={`Clear session settings and revert to default settings`}
           onClick={() => {
-            setColorScheme(false, null);
+            setColorScheme(false, isTeams ? datasetDefault : null);
```

Comment:
I'm not sure I understand the behavior here. Reset means two different things based on the context? How does a Teams user nullify a dataset default? Ideally we add a `selector` that resolves to the reset value (resolution should be opaque/decoupled)

Reply:
Reset always means reset to dataset default (if there is one). We have something else to clear dataset default setting. 

By setting colorScheme to `null`, OSS will load dataset default or color pool from `fo.app_config` or `session.config` from the session. But teams can't update through the session. Therefore, I am pointing the setting to datasetDefault here for TEAMS.

Reply:
Ok. In my routing work, the color scheme will update through a `session` regardless of what the means to the context (Teams, OSS, or a hypothetical). I'm just advocating that this doesn't leak into OSS and component logic